src/akamai/js/akamaiHD_flash.js

```diff
@@ -667,13 +709,26 @@ require("../../../html5-common/js/utils/constants.js");
         break;
       }
       return true;
-    }, this);
+    }
+
   };
+
   /************************************************************************************/
   // Helper methods
   /************************************************************************************/
 
   /**
+   * Returns the SWF instance present in the Dom matching the provided id
+   * @private
+   * @method OoyalaAkamaiHDFlashVideoWrapper#getSwf
+   * @param {string} thisId the id of the swf object sought.
+   * @returns {object} the object containing the desired swf.
+   */
+  var getSwf = function (thisId) {
+    return document.getElementsByName(thisId)[0];
+  };
```

Comment:
if thisId is supposed to be the ID attribute of a DOM element, you want to use document.getElementById. Otherwise the naming is just a little misguiding but otherwise alright.